ui/bin/opensnitch-ui

```diff
@@ -40,11 +40,13 @@ if dist_path not in sys.path:
 from opensnitch.service import UIService
 from opensnitch.config import Config
 from opensnitch.utils import Themes, Utils, Versions
+from opensnitch.utils.xdg import xdg_runtime_dir
 import opensnitch.ui_pb2
 from opensnitch.ui_pb2_grpc import add_UIServicer_to_server
 
 def on_exit():
     server.stop(0)
+    lockfile.unlock()
```

Comment:
on my system this complains about `lockfile is not defined`
a missing import?

Reply:
That's strange, it works for me without any problem.

`lockfile` is defined in line 99 https://github.com/evilsocket/opensnitch/blob/d003cc162c56db1d69bd62265ef3b5216dac4a05/ui/bin/opensnitch-ui#L99, directly after `app`.

What versions (python, qt) are you using?

Reply:
nah, nevermind, my local branch of the PR was not updated, sorry.